caom2-compute/src/main/java/ca/nrc/cadc/caom2/compute/PolygonUtil.java

```diff
@@ -920,13 +910,13 @@ private static Vertex findNearest(Vertex v, List<Polygon> polys)
         Circle minSpanCircle;
     }
     
-    // used by Polygon
-    static PolygonProperties computePolygonProperties(Polygon poly)
+    // used by MultiPolygon
+    static PolygonProperties XXXcomputePolygonProperties(MultiPolygon poly)
```

Comment:
yes, this was redundant and I was trying to see what else to change when I removed it. Will clean up.